### PR TITLE
Feat: Add required maintenance files

### PR DESCRIPTION
**Pull request by JuanDavidBuitrago**
Description:

This PR add maintenance files for the plugin

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+.*.swp
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
```
Comment (mariajgrimaldi) [inline .gitignore]: add end of file

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+universal = 1
+
+[pycodestyle]
+ignore=E501, E722
+exclude=.git,.tvm
+
+[tool:pytest]
+norecursedirs = strains .tvm
+
+[bumpversion:file:tutorcodejail/__init__.py]
```
Comment (mariajgrimaldi) [inline setup.cfg:23]: Did it work?

Comment (JuanDavidBuitrago) [reply setup.cfg:23]: Yes, it did

File: .gitignore (hunk 1 above)
Comment (JuanDavidBuitrago) [reply .gitignore]: Done

**Review by johanseto — APPROVED** [review]

**Review by mariajgrimaldi — APPROVED** [review]

**Pull request merged**